### PR TITLE
Fix RSpec/RepeatedExample and RSpec/RepeatedDescription

### PR DESCRIPTION
**Pull request by backus**
I think some of the repeated description tests could also be removed without reducing coverage, but I just focused on killing the offenses

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] All tests(`rake spec`) are passing.
* [x] The new code doesn't generate RuboCop offenses that are checked by `rake internal_investigation`.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: spec/rubocop/cop/lint/literal_as_condition_spec.rb [hunk 1]
```diff
@@ -138,13 +138,6 @@
       RUBY
       expect(cop.offenses.size).to eq(1)
     end
-
-    it "registers an offense for `not #{lit}`" do
-      inspect_source(<<-RUBY.strip_indent)
-        !#{lit}
```
Comment (pocke) [inline spec/rubocop/cop/lint/literal_as_condition_spec.rb]: We should not remove this test case, it should be `not(#{lit})`. Sorry, this test is my mistake.

**Comment by backus** [comment]
Alright I’ll update that

File: spec/rubocop/cop/lint/literal_as_condition_spec.rb (hunk 1 above)
Comment (backus) [reply spec/rubocop/cop/lint/literal_as_condition_spec.rb]: Fixed

**Comment by bbatsov** [comment]
👍

**Pull request merged**